admiral/debug_functions.sqf

```diff
@@ -287,6 +287,61 @@ adm_debug_fnc_updateLineMarker = {
     DEBUG("admiral.debug",FMT_3("Updated line marker '%1' position between '%2' and '%3'.",_markerName,_posFrom,_posTo));
 };
 
+adm_debug_fnc_createDebugCounterMarkers = {
+    FUN_ARGS_2(_side,_xPos);
+
+    PVT_2(_marker,_markerType);
```

Comment:
Always use `private` in the smallest scope possible. These two are only used in the `foreach`, so it should be in the `foreach`.